haiku-misc/vcardpeople/vcardpeople-1.0.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Utilities to convert between VCard and BeOS 'Person' files"
+DESCRIPTION="The VCard format is an open standard used for storing and \
+interchanging personal (\"Business Card\") data by applications on many \
+platforms. The two small programs in this package will convert between this \
+format and BeOS 'Person' files, making it possible to share contact data \
+between BeOS and other platforms."
+HOMEPAGE="http://goodeveca.net/beos/"
+COPYRIGHT="2002 Peter J. Goodeve"
+LICENSE="VCard"
+REVISION="1"
+SOURCE_URI="http://goodeveca.net/beos/vcardpeople.zip"
+CHECKSUM_SHA256="ec46db14a069dbcec2d524b773b3d8043f056c77c1d7cccce8b6ce6504738f31"
+SOURCE_DIR="VCard-People"
```

Comment:
Since the name of the zip archive does not include any version, I think it would be nice to insert a `SOURCE_FILENAME` parameter (between `CHECKSUM_SHA256` and `SOURCE_DIR`) to have haikuporter rename the zip file after the download:
```
SOURCE_FILENAME="vcardpeople-$portVersion.zip"
```

BTW, do you think `PATCH()` look nicer with an extra tab in two of its middle lines?
```
PATCH()
{
	sed -i \
		-e "s|/boot/develop/etc/makefile-engine|`findpaths -r \
			"makefile_engine" \
			B_FIND_PATH_DEVELOP_DIRECTORY`/etc/makefile-engine|" \
		SOURCE/*/Makefile
}
```
(I only though about this after my previous review.)

Reply:
Or you could move the entire `findpaths` line into an variable, then it would looks much better.

For `SOURCE_FILENAME`, the <portname>-<portversion> format is [required][0] by our policy

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name